README.md

```diff
@@ -126,7 +137,7 @@ equation solver.
 
  bayesian_result_abc = abc_inference(prob1, Tsit5(), t, data, priors)
 ```
-### Using save_idxs for using latent variables 
+### Using save_idxs to declare observables
 
```

Comment:
you might want text here. Describe what the problem is, how to fix it, then show code.